etna/transforms/decomposition/deseasonal.py

```diff
@@ -15,6 +15,8 @@
 from etna.transforms.base import OneSegmentTransform
 from etna.transforms.base import ReversiblePerSegmentWrapper
 
+_DEFAULT_FREQ = object()
```

Comment:
I thought it doesn't make sense to set default `freq=None`, because currenlty `None` is a valid value and we should have some other value for the default.

Reply:
We check that `self._seasonal` is set. They both set in `fit`, so I'm not sure we should check both `_seasonal` and `_freq`.